src/View/Helper/SecureFieldTokenTrait.php

```diff
@@ -55,7 +55,8 @@ protected function _buildFieldToken($url, $fields, $unlockedFields = [])
         $hashParts = [
             $url,
             serialize($fields),
-            $unlocked
+            $unlocked,
+            session_id(),
```

Comment:
Was Security Component requireing native PHP sessions before? If not, could the session id be fetched via https://api.cakephp.org/3.5/class-Cake.Network.Session.html#_id for more flexibility?

Reply:
it's the same thing:

https://api.cakephp.org/3.5/source-class-Cake.Network.Session.html#456

Reply:
I would have preferred a wrapped call, too...